frontend/src/metabase/parameters/components/ParameterValueWidgetTrigger.tsx

```diff
@@ -43,15 +45,15 @@ function ParameterValueWidgetTriggerInner(
   }
 
   return (
-    <div
+    <button
```

Comment:
@romeovs there is `UnstyledButton` in `metabase/ui`

Reply:
I've pushed a commit that does this, but it seems impossible to appease TypeScript for this case so I added a hacky cast. Let me know if you think that's too hacky or if it's acceptable here.